toolkit/scripts/pkggen.mk

```diff
@@ -89,13 +89,45 @@ $(specs_file): $(chroot_worker) $(BUILD_SPECS_DIR) $(build_specs) $(build_spec_d
 		$(if $(TARGET_ARCH),--target-arch="$(TARGET_ARCH)") \
 	        --output $@
 
+grapher_cycle_flags :=
+
+grapher_extra_flags :=
+
+ifeq ($(USE_PREVIEW_REPO),y)
+grapher_extra_flags += --use-preview-repo
+endif
+
+ifeq ($(RESOLVE_CYCLES_FROM_UPSTREAM),y)
+grapher_cycle_flags += --resolve-cycles-from-upstream
```

Comment:
If you decide to keep `grapher_extra_flags`, let's be consistent and do `grapher_extra_flags +=` here instead of `grapher_cycle_flags +=`.